cfme/automate/buttons.py

```diff
@@ -29,15 +52,27 @@ def is_displayed(self):
 class ButtonFormCommon(AutomateCustomizationView):
 
     class options(PotentiallyInvisibleTab):  # noqa
-        type = BootstrapSelect('button_type')
-        # TODO: Add the Ansible fields.
+        form = ConditionalSwitchableView(reference="type")
+        type = VersionPick({
+            Version.lowest(): FakeWidget(read_value='Default'),
```

Comment:
**REQUIRED:** Can you please explain how this Fake Widget works?

Reply:
@psav There is an implementation of it in widgetastic_mangeiq.py. This widget can return any value which provided to `read_value` argument. I need such widget here because `type` is absent in CFME5.8, but `ConditionalSwitchableView` requires that referenced widget provide some value.